tests/integration/standard/test_rate_limit_exceeded.py

```diff
@@ -4,13 +4,14 @@
 from cassandra.cluster import Cluster
 from cassandra.policies import ConstantReconnectionPolicy, RoundRobinPolicy, TokenAwarePolicy
 
-from tests.integration import PROTOCOL_VERSION, use_cluster
+from tests.integration import PROTOCOL_VERSION, use_cluster, scylla_only
 
 LOGGER = logging.getLogger(__name__)
 
 def setup_module():
     use_cluster('rate_limit', [3], start=True)
 
+@scylla_only
```

Comment:
Can we have explanation why to run it only on scylla ?

Reply:
RateLimitExceededException is a scylla only feature... it's error only scylla generate, and cassandra doesn't